app/styles/card.css

```diff
@@ -0,0 +1,53 @@
+/* INFO: not adding width/max-width to let it decicde by layout */
+:root {
+  --border-color: #00000040;
+  /* TODO: get primary color from variables.css, once primary color is decided  */
+  --primary: #1d1283;
+}
+.card {
+  max-width: 41rem;
+  padding: 1.5rem 1rem;
+  border: 1px solid var(--border-color);
+  border-radius: 3rem;
```

Comment:
border-radius is too much I think we can do 1 rem here

Reply:
It as per desgin, please check [Figma Desgin](https://www.figma.com/proto/xKTlVbkqJSd3wZKH3yAAEf/RDS_Onboarding?node-id=0%3A3&scaling=scale-down&page-id=0%3A1)